src/init.ts

```diff
@@ -148,48 +165,72 @@ export class NightwatchInit {
         answers.remoteEnv.access_key = 'SAUCE_ACCESS_KEY';
       }
 
-      if (!answers.remoteBrowsers) {
-        if (answers.browsers) {
-          // Copy all browsers, except selenium-server (if present)
-          answers.remoteBrowsers = [...answers.browsers].filter((browser) => browser !== 'selenium-server');
-        } else {
-          answers.remoteBrowsers = BROWSER_CHOICES.map((browser) => browser.value);
+      if (!this.options.mobile) {
+        if (!answers.remoteBrowsers) {
+          if (answers.browsers) {
+            // Copy all browsers, except selenium-server (if present)
+            answers.remoteBrowsers = [...answers.browsers].filter((browser) => browser !== 'selenium-server');
+          } else {
+            answers.remoteBrowsers = BROWSER_CHOICES.map((browser) => browser.value);
+          }
         }
+  
+        // If backend is only remote (no local), delete answers.browsers (if present)
+        // and set the defaultBrowser.
+        if (!backendHasLocal) {
+          if (answers.browsers) {
+            delete answers.browsers;
+          }
+          answers.defaultBrowser = answers.remoteBrowsers[0];
+        }
+      } else {
+        answers.remoteBrowsers = [];
+        answers.browsers = [];
```

Comment:
What if the user wants to test on both mobile and desktop browsers? It will make the desktop browsers to empty array in that case also.

Reply:
We need to rethink the structure of `refineAnswers`.

Reply:
So this is case, when user will pass `--mobile` option with `npm init nightwatch` command then we will only display the mobile related questions.